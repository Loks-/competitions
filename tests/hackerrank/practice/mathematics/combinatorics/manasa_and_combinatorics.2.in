5
73139
87746
25679
95483
73883
