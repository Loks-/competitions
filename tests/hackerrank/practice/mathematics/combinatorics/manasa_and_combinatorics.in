7
1
2
73139
87746
25679
95483
73883
