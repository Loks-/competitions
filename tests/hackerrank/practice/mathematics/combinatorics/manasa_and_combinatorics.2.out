0
0
35199
0
0
